src/main/include/subsystems/Climber.h

```diff
@@ -0,0 +1,25 @@
+#include <rev/SparkMax.h>
```

Comment:
Your guard against multiple inclusion is missing:

```suggestion
#pragma once

#include <rev/SparkMax.h>
```

Reply:
added pragma once to rev/sparkmax